main.go

```diff
@@ -185,6 +186,15 @@ func populateArgs(c *cli.Context) (
 	return
 }
 
+func stringify(input any) string {
+	inputBytes, err := json.Marshal(input)
+
+	if err != nil {
+		logger.Errorf("Error in stringify %v", err)
```

Comment:
We should log it as Warning, if it's not a fatal error.

Reply:
done.. changed it to warn